### DISCUSSION TITLE
Help please:installing on Mac (and use XCode?)

### DISCUSSION THREAD

**Discussion by Caliena** [SWivid/F5-TTS#323] [Q&A]
I am still very much a a beginner to coding and using github. 

1. How do I install to my MacBook Pro (macOS Sonoma 14.4.1)?
2. Can I use XCode? 
3. Will I need anything else? 
4. This is what I have so far.  Am I on the right track? 
5. and what do I do next to use the F5-TTS?

<img width="1415" alt="Screenshot 2024-10-29 at 10 11 41 PM" src="https://github.com/user-attachments/assets/4de9d493-945c-4b9a-bf61-1dac50d11533">

**Comment by SWivid** [comment]
Hi @Caliena , might check for pinokio usage if not familiar with code stuff.

There are excellent tutorials on youtube for usage, check it out.
[pinokio e2-f5-tts](https://pinokio.computer/item?uri=https://github.com/pinokiofactory/e2-f5-tts)

**Reply by haidnt** [reply]
> Hi @Caliena , might check for pinokio usage if not familiar with code stuff.
> 
> There are excellent tutorials on youtube for usage, check it out. [pinokio e2-f5-tts](https://pinokio.computer/item?uri=https://github.com/pinokiofactory/e2-f5-tts)

Thank you very much bro! I love you 🤩
